### PR TITLE
sentry: Add breadcrumb for Zulip event type (and op when present)

### PR DESCRIPTION
**Pull request by chrisbobbe**

### CODE REVIEW

File: src/events/doEventActionSideEffects.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ import { ensureTypingStatusExpiryLoop } from '../typing/typingActions';
  * To be dispatched before the event actions are dispatched.
  */
 export default (action: EventAction): ThunkAction<Promise<void>> => async (dispatch, getState) => {
+  addBreadcrumb({
+    category: 'redux-action.zulip-event',
```
Comment (chrisbobbe) [inline src/events/doEventActionSideEffects.js:16]: Could shrink this down to `action.event` to reduce payload size maybe?

**Comment by gnprice** [comment]
Thanks!

I think I'd prefer to express this information in the Zulip API's terms -- that feels a bit more stable, as it's an interface that's already constrained by compatibility and isn't subject to our refactoring at will.

I'd also like to include the event's `op` property, when present, as different values of that can send us down a totally different codepath in much the same way as `type`. That will also be simplified by making it in terms of the event.

I think the way to do that is probably to move this breadcrumb up to the caller of `doEventActionSideEffects`, in `handleEvent`. It's true that this is a side effect of sorts… but it's purely in logging, which is a kind of side effect that we regularly do in contexts that are otherwise pure, like reducers. So I don't think there's a need for this to be grouped with the more visible side effects.

**Comment by chrisbobbe** [comment]
Thanks for the review! Revision pushed.

**Comment by gnprice** [comment]
Thanks for the revision! Looks good -- merging.

**Pull request merged**